ci/tsqa/tests/test_forward_proxy.py

```diff
@@ -0,0 +1,123 @@
+'''
```

Comment:
If at all possible, can you convert this to the new AU test system? @dragon512 can perhaps assist as well.